client/packages/core/package.json

```diff
@@ -33,6 +30,8 @@
     "vitest": "^1.6.0"
   },
   "dependencies": {
+    "@types/object-hash": "^3.0.3",
+    "@types/uuid": "^8.3.4",
```

Comment:
I stopped exporting the uuid class in https://github.com/instantdb/instant/pull/48/files, so we might not need `@types/uuid` in `dependencies`

Reply:
Nice, I think you're right!  We're not importing `uuid` in any of the `.d.ts` files.